core/patina_debugger/src/arch/x64.rs

```diff
@@ -190,8 +191,25 @@ impl DebuggerArch for X64Arch {
                     asm!("mov {0}, cr3", "mov cr3, {0}", out(reg) _, options(nostack, nomem));
                 }
             }
+            Some("mtrr") => {
+                if let Some(val) = tokens.next() {
+                    let mtrr = patina_mtrr::create_mtrr_lib(0);
+                    let addr = match u64::from_str_radix(val.trim_start_matches("0x"), 16) {
+                        Ok(a) => a,
+                        Err(_) => {
+                            let _ = out.write_str("Invalid address format");
```

Comment:
Something like this would be a little more helpful.

```suggestion
                            let _ = write!(out, "Invalid address format: '{}'. Expected hex address (e.g., 0x1000)", val);
```